docs/src/tutorials/getting-started-vscode/README.md

```diff
@@ -0,0 +1,13 @@
+# My first Visual Studio Code project
+
+This tutorial is recommended for all users who are starting a new
+project in Visual Studio Code for the first time and will go through
+some small example R and Python code.
+
+By the end of this tutorial you should have in place a [Conda
```

Comment:
I think documenting conda environment creation would be useful later, but as it's not here yet could just have something like you'll have in place a basic vscode setup like on your computer.